release-notes/5.0/preview/api-diff/preview2/.Net/5.0-preview2_System.Diagnostics.md

```diff
@@ -0,0 +1,11 @@
+# System.Diagnostics
+
+``` diff
+ namespace System.Diagnostics {
+-    public class Process : Component {
++    public class Process : Component, IDisposable {
```

Comment:
This is a strange diff.  `Component` is `IDisposable`, so `Process` was already implementing `IDisposable`.

Reply:
I think its not checking the transitive changes. we changed the api defination https://github.com/dotnet/runtime/pull/31671/files#diff-d3273f4291d481a6dc45193541efbb6aR30